images/linux/ubuntu1804.json

```diff
@@ -323,6 +314,22 @@
                 "RUN_VALIDATION={{user `run_validation_diskspace`}}"
             ]
         },
+        {
+            "type": "shell",
+            "scripts":[
+                "{{template_dir}}/scripts/installers/post-deployment.sh"
+            ],
+            "execute_command": "sudo sh -c '{{ .Vars }} {{ .Path }}'"
+        },
+        {
+            "type": "shell",
+            "inline": [
+                "rm -rf {{user `helper_script_folder`}}",
```

Comment:
Is it possible to move these lines to post-deployment?

Reply:
Done. Thank you.